README.md

```diff
@@ -390,7 +390,7 @@ One of our main goals with open sourcing the Svix dispatcher is ease of use. The
 This is why this code has been adjusted before being released, and some of the features, optimizations, and behaviors supported by the hosted dispatcher are not yet available in this repo. With that being said, other than some known incompatibilities, the internal Svix test suite passes. This means they are already mostly compatible, and we are working hard on bringing them to full feature parity.
 
 # Re-driving Redis DLQ
-We have an undocumented endpoint for re-driving failed messages that are DLQ'ed. You can do this by calling `POST /api/v1/admin/redrive-dlq/`.
+We have an undocumented endpoint for re-driving failed messages that are DLQ'ed. You can do this by calling `PUT /api/v1/admin/redrive-dlq/`.
```

Comment:
It was originally, but the power of @svix-gabriel's rhetoric persuaded me otherwise: https://github.com/svix/svix-webhooks/pull/1478#discussion_r1790909322

Reply:
Bleh ignore me. I got confused. Yeah you're right, I had it backward from a prior change.